moped-editor/src/views/projects/NewProjectView.js

```diff
@@ -222,15 +226,16 @@ const NewProjectView = () => {
 
     //data from ProjectTeamTable going to database
     let teamData = _.toArray({ ...StaffRows });
-    _.forEach(teamData, function(value) {
+     _.forEach(teamData, function(value) {
       let name_array = value.name.name;
       let name_split = name_array.split(" ");
       let first_name = name_split[0];
       let last_name = name_split[1];
       let workgroup = value.workgroup;
       let role_name = value.role;
-      addStaff({ variables: { workgroup, role_name, first_name, last_name } });
-    });
+      let notes = value.notes.userInput;
+      addStaff({ variables: { workgroup, role_name, first_name, last_name, notes } });
+     });
```

Comment:
The other devs seem ok with using lodash, I do have a recommendation to import specific libraries from lodash as opposed to the entire library. Not a big deal, if it is hard to refactor it is not important.